packages/pubsub/lib/trace.ts

```diff
@@ -0,0 +1,5 @@
+import tracer from 'dd-trace';
+
+tracer.init({
+    service: 'nango-pubsub'
+});
```

Comment:
it has no support for activemq? 😢

Reply:
last time I checked amqp it was still good on 0.9 but support for 1.0 was almost inexistant, not sure which one activemq is using

Reply:
https://activemq.apache.org/components/classic/documentation/amqp